src/src/hilell/hw/FileManager.java

```diff
@@ -0,0 +1,108 @@
+package hilell.hw;
+
+import java.io.IOException;
+import java.nio.file.*;
+
+
+public class FileManager {
+    private static Path currentDirectory;
+
+    public static void main(String[] args) {
+        currentDirectory = Paths.get("C:\\work\\Java_Pro");
```

Comment:
Please keep in mind that hardcoding a path is not recommended. 
The better approach would be to get the root path using this snippet: `System.getProperty("user.home");`, this should return the root of your drive (C://). With this, you can start your command line exactly there and then run all your operations.